WireAuthentication/Sources/WireAuthentication/Components/LoginViaEmailComponent.swift

```diff
@@ -87,12 +86,18 @@ class LoginViaEmailComponent: Component<LoginViaEmailComponentDependency> {
             loginViaEmailUseCase: loginViaEmailUseCase,
             backendEnvironment: backendEnvironment,
             email: email,
-            accountsURL: dependency.accountsURL,
             passwordValidator: dependency.passwordValidator,
             canCreateAccount: canCreateAccount,
             didDetectDomainConflict: didDetectDomainConflict,
-            onCreateAccount: { [dependency] in
-                dependency?.bridge.sendOutboundEvent(.accountRegistrationRequested)
+            onCreateAccount: { [dependency, backendEnvironment] in
+                guard let dependency else { return }
+                dependency.router.dismissSheet()
```

Comment:
The new screens are all in a sheet on the root view. If we don't dismiss the sheet before navigating to the registration flow (outside of this module) then the sheet will remain visible above the registration flow.